src/core_plugins/kibana/public/dashboard/top_nav/show_options_popover.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import './options_popover.less';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { OptionsMenu } from './options';
+
+import {
+  EuiWrappingPopover,
+} from '@elastic/eui';
+
+let isOpen = false;
+
+const container = document.createElement('div');
+
+const onClose = () => {
+  ReactDOM.unmountComponentAtNode(container);
+  isOpen = false;
+};
+
+export function showOptionsPopover({
+  anchorElement,
+  darkTheme,
+  onDarkThemeChange,
+  useMargins,
+  onUseMarginsChange,
+  hidePanelTitles,
+  onHidePanelTitlesChange,
+}) {
+  if (isOpen) {
```

Comment:
Maybe consider renaming `showOptionsPopover` to `toggleOptionsPopover`? If I'm reading this line correctly. It sometimes shows, sometimes hides, depending on the value of isOpen.

Also, you might just make this function body something like this, for consistency:

```js
export function toggleOptionsPopover(props) {
  if (isOpen) {
    closeOptionsPopover();
  } else {
    openOptionsPopover(props);
  }
}
```

*shrugs* Not important.